docs/channel/README.md

```diff
@@ -0,0 +1,224 @@
+# Knative Eventing Kafka Channel
+
+## Motivation for the channel
+
+This repository contains another KafkaChannel implementation, which we believe is more flexible and scalable.
+
+There are already 2 other Kafka Channel implentations:
+the [consolidated KafkaChannel](https://github.com/knative-sandbox/eventing-kafka/blob/main/pkg/channel/consolidated/README.md)
+and
+the [distributed KafkaChannel](https://github.com/knative-sandbox/eventing-kafka/blob/main/pkg/channel/distributed/README.md)
+.
+
+While working on these 2 channel implementations, we realized we could reuse the dataplane for other Knative Kafka
+components, namely Kafka Broker and the Kafka Sink.
```

Comment:
wanna say Knative Kafka Broker - to not confuse this with the actual Apache Kafka broker(s)?